egs/iam/v1/local/chain/tuning/run_cnn_chainali_1b.sh

```diff
@@ -22,6 +22,7 @@ stage=0
 
 nj=30
 train_set=train
+decode_val=true
```

Comment:
is there a reason there are no results shown herE?

Reply:
I started the training scripts, I will surely update the result today.